tbump/git_bumper.py

```diff
@@ -45,11 +45,12 @@ def get_current_branch(self):
         return out
 
     def get_tracking_ref(self):
+        branch_name = self.get_current_branch()
         rc, out = self.run_git(
             "rev-parse", "--abbrev-ref", "--symbolic-full-name", "@{u}",
             raises=False)
         if rc != 0:
-            ui.fatal("Failed to get tracking ref")
+            ui.fatal("Current branch (%s)" % branch_name, "does not track anything. Cannot push.")
```

Comment:
Neat, I just happened to run into that error today.

Although I wish it would just let me continue without giving the option to push when there's no remote branch, instead of flat out refusing to create the tag and giving up. The nice thing about Git is you can work locally and only push a branch when it's ready to be published :]

Reply:
Ah. Now I remember why I needed real exceptions :)

I'll do that in an other commit

Reply:
Awesome, thanks!